src/exclusive-or.rst

```diff
@@ -111,91 +110,89 @@ bits. Hence the name, *bitwise* XOR.
    \end{aligned}
 
 .. [#binary-integer]
-   Usually, numbers are already stored in binary internally, so this
-   doesn't actually take any work. When you see a number prefixed with
-   “0b”, the remaining digits are a binary representation.
+   Usually, numbers are internally stored in binary, so this
+   does not take any work. When a number is prefixed with
+   “0b” it means that the remaining digits are a binary representation.
 
 One-time pads
 ~~~~~~~~~~~~~
 
 XOR may seem like an awfully simple, even trivial operator. Even so,
-there's an encryption scheme, called a one-time pad, which consists of
-just that single operator. It's called a one-time pad because it
-involves a sequence (the “pad”) of random bits, and the security of the
-scheme depends on only using that pad once. The sequence is called a pad
+there is an encryption scheme. It is called a one-time pad, which consists of
+a single operator with a sequence (“pad”) of random bits. The security of the
+scheme depends on using the pad only once. The sequence is called a pad
 because it was originally recorded on a physical, paper pad.
 
-This scheme is unique not only in its simplicity, but also because it
-has the strongest possible security guarantee. If the bits are truly
-random (and therefore unpredictable by an attacker), and the pad is only
-used once, the attacker learns nothing about the plaintext when they see
+The scheme is unique not only in its simplicity. It
+has the highest security guarantee possible. If the bits are truly
+random, they become unpredictable for an attacker. Additionally, if the pad is only
+used once, the attacker learns nothing about the plaintext when viewing
 a ciphertext. [#msg-exists]_
 
 .. [#msg-exists]
-   The attacker does learn that the message exists, and, in this simple
-   scheme, the length of the message. While this typically isn't too
-   important, there are situations where this might matter, and there
-   are secure cryptosystems to both hide the existence and the length of
+   The attacker does learn that the message exists and the message length
+   in this simple scheme. While this typically is not too
+   important, there are situations where this matters. 
+   Secure cryptosystems exist to both hide the existence and the length of
    a message.
 
 
-Suppose we can translate our plaintext into a sequence of bits. We also
-have the pad of random bits, shared between the sender and the (one or
-more) recipients. We can compute the ciphertext by taking the bitwise
-XOR of the two sequences of bits.
+Suppose we translate our plaintext into a sequence of bits. We 
+have the pad of random bits that are shared between the sender and the recipients (one or
```

Comment:
"translate our plaintext into a sequence of bits" doesn't give the pad to the sender and the recipient.
also that needs to be assumed